2022-04-20 06:29:13.447 | INFO     | db:__init__:23 - Datebase Class Created With <module 'mysql' from '/usr/local/lib/python3.9/dist-packages/mysql/__init__.py'>
2022-04-20 06:30:49.007 | DEBUG    | main:startScrapperSchdular:38 - startScrapper
2022-04-20 06:30:49.061 | ERROR    | main:Error:57 - 
 HTTPException Error : 500 Internal Server Error: The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application. 

2022-04-20 06:30:49.061 | ERROR    | main:error:69 - 
Error(name, code)->-----------name Internal Server Error   code 500

